sys/net/include/etx_beaconing.h

```diff
@@ -21,27 +21,30 @@
 #define ETX_BEACONING_H_
 #include <stdint.h>
 #include "sixlowpan.h"
-
+/**
+  * makes a function-name in C++ have 'C' linkage so that client C code can link to (i.e use) your function using a 'C' compatible header  file that contains just the declaration of your function
+  */
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-//[option|length|ipaddr.|packetcount] with up to 15 ipaddr|packetcount pairs
-// 1 Byte 1 Byte  1 Byte  1 Byte
+/**[option|length|ipaddr.|packetcount] with up to 15 ipaddr|packetcount pairs
+  *   1 Byte 1 Byte  1 Byte  1 Byte 
+  */
 #define ETX_BUF_SIZE   (32)
-
 #define ETX_RCV_QUEUE_SIZE     (128)
 
-/*
- * Default 40, should be enough to get all messages for neighbors.
- * In my tests, the maximum count of neighbors was around 30-something
- */
+/** Stacksizes are huge in debug mode, so memory is rare so we fix it at 15
+  * Default 40, should be enough to get all messages for neighbors.
+  * In my tests, the maximum count of neighbors was around 30-something
+  */
 #if ENABLE_DEBUG
-#define ETX_MAX_CANDIDATE_NEIGHBORS (15) //Stacksizes are huge in debug mode, so memory is rare
+#define ETX_MAX_CANDIDATE_NEIGHBORS (15) 
 #else
 #define ETX_MAX_CANDIDATE_NEIGHBORS (40)
 #endif
-//ETX Interval parameters
+/**ETX Interval parameters
+  */
```

Comment:
@authmillenon According to the examples at http://www.stack.nl/~dimitri/doxygen/manual/grouping.html#memgroup `@name` should be placed outside and above the braces..